BWEnv/src/zmq_server.cc

```diff
@@ -375,7 +390,10 @@ std::vector<int8_t> ZMQ_server::handleCommands(const torchcraft::fbs::Commands*
   return results;
 }
 
-int ZMQ_server::getPort()
-{
+int ZMQ_server::getPort() {
   return port;
 }
+
+std::string ZMQ_server::getFileSocketName() {
```

Comment:
Nit: `getSocketFileName()`? Could also use http://en.cppreference.com/w/cpp/experimental/fs/path/filename

Reply:
Ah sorry, it refers to `file_socket` so the name should be fine :)